qiskit/visualization/utils.py

```diff
@@ -160,21 +161,91 @@ def get_param_str(op, drawer, ndigits=3):
     return param_str
 
 
-def get_bit_label(drawer, register, index, qubit=True, layout=None, cregbundle=True):
+def get_bits_regs_map(circuit, bits, cregbundle):
+    """Map the bits and registers to the index from the top of the drawing.
+    The key to the dict is either the (Qubit, Clbit) or if cregbunle True,
```

Comment:
```suggestion
    The key to the dict is either the (Qubit, Clbit) or if cregbundle True,
```

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/7319/commits/55e62189a965a13f2fee8bdb9cdfb1f9cb88d3fd.